docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc

```diff
@@ -77,13 +80,13 @@ e.g.
 $ curl -v http://localhost:8080/openapi?format=JSON
 < HTTP/1.1 200 OK
 ...
-{"openapi": "3.0.1" ... }
+{"openapi": "{OpenAPIVersion}.0" ... }
```

Comment:
FWIW for some reason this doesn't render on GitHub - https://github.com/wildfly/wildfly/blob/44db5eefc1b340055f7b9e45cf23e947f99cf2df/docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc#https-endpoint - but that doesn't matter.